tools/misc/tpm2_checkquote.c

```diff
@@ -267,7 +267,7 @@ static bool parse_selection_data_from_file(FILE *pcr_input,
     }
 
     if (le64toh(pcrs->count) > ARRAY_LEN(pcrs->pcr_values)) {
-        LOG_ERR("Malformed PCR file, pcr count cannot be greater than %zu, got: %zu",
+        LOG_ERR("Malformed PCR file, pcr count cannot be greater than %zu, got: %llu",
```

Comment:
The C standard only guarantees a minimal precision for integer types and so the specifier may not be portable for a fixed-width type.
```suggestion
        LOG_ERR("Malformed PCR file, pcr count cannot be greater than %zu, got: %" PRIu64 " ",
```

Reply:
I wasn't sure whether we use these kinda ugly PRIu64 - but we do, so then yes this is more correct here.